src/project/manifest/metadata.rs

```diff
@@ -50,4 +50,7 @@ pub struct ProjectMetadata {
 
     /// URL of the project documentation
     pub documentation: Option<Url>,
+
+    /// URL of the pypi name mapping
+    pub pypi_name_mapping: Option<Url>,
```

Comment:
Im thinking that maybe we should have a mapping per channel. So you can use the prefix one for `conda-forge` but use a custom mapping for your own private channel?

WDYT?

Reply:
good idea! 
some small discovery questions about how do you see it:
* do we want to also allow to override conda-forge with your own mapping?
* do we want to expose configuration via `pypi-name-mapping = {"conda-forge": ..my_path, "my-own-channel": ..my_another_path } `, or to hide the channel in the mapping itself? ( I go with the first approach )

Reply:
I like this, although it's a table. Could we move it into its own table?
e.g.:
```toml
[pypi-name-mapping]
conda-forge = "URL"
"https://prefix.dev/my_channel" = "mapping_url"
```

Reply:
I'm having some doubts on the name. What about:
- `conda-pypi-map`
- `channel-index-map`
- `conda-pypi-package-map`

Reply:
What about starting a `[pypi-options]` table? I feel there could be a number of future options to handle (keyring, extra index, etc.)

Not sure re. the specific name of the map, but we could maybe take 'pypi' out of it if it sits under a  `[pypi-options]` table

Reply:
I agree with @nichmor, not a bad idea but not required in this case.